internal/pkg/aws/ec2/ec2.go

```diff
@@ -95,6 +106,20 @@ func (c *EC2) GetSecurityGroups(filters ...Filter) ([]string, error) {
 	return securityGroups, nil
 }
 
+func (c *EC2) subnets(filters ...Filter) ([]*ec2.Subnet, error) {
```

Comment:
nit: Should the function name contain a capital S for Subnets?

Reply:
I like what Wanxian has actually where there is a `SubnetIDs` method that's public which returns `([]string, error)`. Then the IDs can be used to look up other details.

If we were to make this method public then there would be a need for customers to import both our `internal/aws/ec2` pkg but also the aws sdk's `ec2` pkg because of the return type.

Reply:
That's fair, just curious as to if it would be useful for customers to be able to use this method or if SubnetIDs is sufficient?